### PR TITLE
Add cloud instance

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/10119

source https://www.microsoft.com/en-ww/microsoft-365/roadmap?filters=&searchterms=95128

### CODE REVIEW

File: Teams/adobe-acrobat-pdf-viewer-teams.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ Know the following information about this functionality:
 * After the policy is set up, it typically [takes a few hours](teams-app-setup-policies.md#considerations-and-limitations) for the app to be available for users.
 * The PDF files that are pinned as tabs in channels and the PDF files available in the Assignments app continue to open in the native viewer of Teams and not in the Adobe Acrobat app.
 * Adobe Acrobat as a default PDF viewer in Teams works only on desktop and web clients. It isn't supported on the mobile client.
+* Currently available for Worldwide (Standard Multi-Tenant) cloud instance only.
```
Comment (JohanFreelancer9) [inline Teams/adobe-acrobat-pdf-viewer-teams.md:77]:
```suggestion
* Currently available for worldwide (standard multitenant) cloud instances only.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by scanum — APPROVED** [review]

File: Teams/adobe-acrobat-pdf-viewer-teams.md (hunk 1 above)
Comment (get-itips) [reply Teams/adobe-acrobat-pdf-viewer-teams.md:77]: Author requests that I use "Commercial cloud"

**Review by ashishguptaiitb — APPROVED** [review]
LGTM.

**Comment by scanum** [comment]
@ashishguptaiitb 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by ashishguptaiitb** [comment]
Hi @scanum, I can't merge. Can you do it for me?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by get-itips** [comment]
@tiburd

**Pull request merged**